js/script-loader.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// eslint-disable-next-line no-var
+var loadScript = (function () {
+  const cache = new Map();
+
+  return function (path) {
+    if (!path.includes('://')) {
+      path = chrome.runtime.getURL(path);
```

Comment:
this function is used only on own pages so no need to expand the URL.

Reply:
It's not required for caching because we always use the same path as a parameter, because there's no need to use different paths for scripts.

Reply:
Also, why Map and why do you store the script elements? I see no need for that.

Reply:
> It's not required for caching because we always use the same path as a parameter, because there's no need to use different paths for scripts.

Then we should always use absolute path (starting with `/`) as the path parameter.

> why Map and why do you store the script elements? I see no need for that.

Yes a Set is enough.

Reply:
I don't mind explicit `/`. It's just not required currently since our html pages are in the root directory.